specifyweb/specify/tree_views.py

```diff
@@ -56,50 +56,50 @@ def wrapper(*args, **kwargs):
                         "schema": {
                             "type": "array",
                             "items": {
-                                "type": "array",
-                                "items": [
-                                    {
+                                "type": "object",
```

Comment:
This doesn't look right. The endpoint returns an array:
![image](https://user-images.githubusercontent.com/40512816/220391847-15477ecd-282c-4beb-9f63-f7a95089bc4d.png)

Reply:
ya, i thought so too, but was the only way I could get the schema validation test to work.  Feel free to play around with it

Reply:
I originally defined the openapi schema for that endpoint. I could not find any other endpoint on which to model it from. I wanted to provide additional information for each of the items of the nested array. Do you know if there is a way to do this without losing any information?
I am willing to mess around with the schema structure for this endpoint and find a solution.

Reply:
I see what you mean. Looks like it's possible, but though a kind of a work-arround: https://stackoverflow.com/a/57465199/8584605 (the comment uses yaml, but we can write the same in json)

Converting the endpoint to return object is not recommended as that would increase the payload size significantly